### DISCUSSION TITLE
v5.0.1 Developer Preview

### DISCUSSION THREAD

**Discussion by xiaokangwang** [v2fly/v2ray-core#1513] [General]
This is a pre-release version of V2Ray intended for developer preview. Users are not advised to use this version right now. Contributors are encouraged to send pull requests to the V5 version of the codebase.
This release note is being edited to reflect all significant changes made to the codebase, please contact us if you think something is missing.
Most new features are currently undocumented. The documents will be updated over time.

## New Features
* **Jsonv5** New configuration format. This enables support for automatic component recognition. 
    * [ZH Document](https://www.v2fly.org/v5/config/overview.html) is work in progress.
    * [EN Document](https://www.v2fly.org/en_US/v5/config/overview.html)  is work in progress.
    * Thanks @AkinoKaede, @kslr, @mzz2017.
* **Burst Observatory** Measure connection quality to a significant amount of outbound at the same time. Thanks @qjebbs.
* **Multi Observatory** Run more than one Observatory at the same time.
* **Router** Load balancing strategy LeastLoad. Thanks @qjebbs.
* **Jsonv4** Additional representation format for Jsonv4 configure file format. This includes YAML, TOML. Thanks @qjebbs.
* **Jsonv4** General purpose merger for all configure formats. Thanks @qjebbs.
* **Developer Assistance** Protobuf configure format reverse engineering.

<hr /><em>This discussion was created from the release <a href='https://github.com/v2fly/v2ray-core/releases/tag/v5.0.1'>v5.0.1 Developer Preview</a>.</em>

**Comment by luciferyuzhi** [comment]
什么时候发布一个5版本的正式版，哎，openwrt插件passwall的v2ray版本不小心点了升级，现在无法使用了，只能等正式版，哎，出个.4或者.5的正式版就好了
